lang/main-sc.json

```diff
@@ -220,7 +220,7 @@
         "muteParticipantDialog": "Seguru chi boles pònnere custa persone a sa muda? No as a pòdere torrare a ativare s'àudio issoro, però is utentes ddu ant a pòdere ativare in cale si siat momentu.",
         "muteParticipantTitle": "Boles pònnere custa persone a sa muda?",
         "Ok": "AB",
-        "passwordLabel": "Unu partetzipante at blocadu sa riunione. Inserta sa $t(lockRoomPassword) pro intrare.",
+        "passwordLabel": "$t(lockRoomPasswordUppercase)",
```

Comment:
Actually, this in English is:
`"passwordLabel": "The meeting has been locked by a participant. Please enter the $t(lockRoomPassword) to join.",`
Can you change it?